server/handlers/handlers.go

```diff
@@ -616,6 +627,10 @@ func (ah *ActivityHandler) handleReactions(postID, channelID string, isDirectOrG
 }
 
 func (ah *ActivityHandler) handleDeletedActivity(activityIds clientmodels.ActivityIds) string {
+	if ah.plugin.GetSyncNotifications() {
```

Comment:
nit: should we update the notification to let the user know the message has been removed? The permalink we posted will lead nowhere.

Reply:
Yes, we do need to define the behaviour here -- will track as a task.